### PR TITLE
50230_display_user_profile

### PR DESCRIPTION
**Pull request by shakil-fg**

### CODE REVIEW

**Comment by sujoydatta** [comment]
ok

**Comment by mdmahmudurrahman** [comment]
OK

File: app/views/admin/users/index.html.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
     </tr>
     <% @users.each do |user| %>
       <tr>
-        <td><%= link_to user.name, "#" %></td>
+        <td><%= link_to user.name, user_path(user) %></td>
```
Comment (MuhammadTamzid) [inline app/views/admin/users/index.html.erb]: remove ()

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -6,7 +6,7 @@
     </tr>
     <% @users.each do |user| %>
       <tr>
-        <td><%= link_to user.name, "#" %></td>
+        <td><%= link_to user.name, user_path(user) %></td>
```
Comment (MuhammadTamzid) [inline app/views/users/index.html.erb]: remove ()

File: app/views/admin/users/index.html.erb (hunk 1 above)
Comment (MuhammadTamzid) [inline app/views/admin/users/index.html.erb]: remove ()

File: app/views/users/index.html.erb (hunk 2 above)
Comment (MuhammadTamzid) [inline app/views/users/index.html.erb]: remove ()

**Comment by MuhammadTamzid** [comment]
Ok

**Pull request merged**